site_scons/state.py

```diff
@@ -176,6 +176,7 @@ def _setEnvWithDependencies():
             (PathVariable('LOG_LIB', 'log libraries path', os.path.join(env['LOG_DIR'], "lib"), PathVariable.PathIsDir)),
             (PathVariable('PROTOBUF_INC', 'protobuf include path', os.path.join(env['PROTOBUF_DIR'], "include"), PathVariable.PathIsDir)),
             (PathVariable('PROTOBUF_LIB', 'protobuf libraries path', os.path.join(env['PROTOBUF_DIR'], "lib"), PathVariable.PathIsDir)),
+            (PathVariable('LUA_INC', 'lua include path', os.path.join(env['LUA_DIR'], "include"), PathVariable.PathIsDir)),
```

Comment:
Well, do we want to comment each of these variables? I don't really want to make LUA_INC look special in any way, we either comment all of them or none.